### ISSUE TITLE
Add a cover letter for multi-patch Pull Requests

### ISSUE THREAD

**Issue by dscho** [rtyley/submitgit#9]
At the moment, patch series are sent without a cover letter.

It would be *really* great if a patch series consisting of more than one patch could have the initial comment of the Pull Request as a cover letter. Maybe even with the link to the original Pull Request appended at the end of the cover letter?

This would also make for a fine mail to use in the "In-reply-yo" header instead of the mail containing the first patch...

**Comment by rtyley** [comment]
Yep, this on my todo list! I think I agree with everything in your description, including the `In-Reply-To:` header obviously.
